examples/Epidemiology/SEI3HRD_example.jl

```diff
@@ -106,18 +114,22 @@ function run_model(times::Unitful.Time, interval::Unitful.Time, timestep::Unitfu
     epilist = EpiList(traits, abun_v, abun_h, movement, transitions, param)
     rel = Gauss{eltype(epienv.habitat)}()
 
+    # multiple dispatch in action
+    rngtype = stochasticmode ? Random.MersenneTwister : MedianGenerator
+
     # Create epi system with all information
-    epi = EpiSystem(epilist, epienv, rel, scotpop)
+    epi = EpiSystem(epilist, epienv, rel, scotpop; rngtype=rngtype)
 
     # Add in initial infections randomly (samples weighted by population size)
     N_cells = size(epi.abundances.matrix, 2)
-    samp = sample(1:N_cells, weights(1.0 .* human(epi.abundances)[1, :]), 100)
+    samp = sample(ic_rng, 1:N_cells, weights(1.0 .* human(epi.abundances)[1, :]), 100)
     virus(epi.abundances)[1, samp] .= 100 # Virus pop in Environment
     human(epi.abundances)[2, samp] .= 10 # Exposed pop
 
     # Run simulation
     abuns = zeros(Int64, numclasses, N_cells, 366)
     times = 2months; interval = 1day; timestep = 1day
```

Comment:
We don't need 366 timesteps, and these time settings overwrite the function arguments:
```suggestion
    abuns = zeros(Int64, numclasses, N_cells, floor(Int, times / interval) + 1)
```